fury/data/fetcher.py

```diff
@@ -75,41 +75,52 @@ def _already_there_msg(folder):
     print(msg)
 
 
-def _get_file_md5(filename):
-    """Compute the md5 checksum of a file."""
-    md5_data = md5()
-    with open(filename, 'rb') as f:
-        for chunk in iter(lambda: f.read(128 * md5_data.block_size), b''):
-            md5_data.update(chunk)
-    return md5_data.hexdigest()
+def _get_file_sha(filename):
+    """Generates SHA checksum for the entire file in blocks of 256
 
+    Parameters
+    ----------
+    filename: str
+        The path to the file whose sha checksum is to be generated
+    Returns
```

Comment:
docstyle: Can you add an empty line between `Returns` and `The path to the file...`